docs/modules/localstack.md

```diff
@@ -7,7 +7,7 @@ Testcontainers module for [LocalStack](http://localstack.cloud/), 'a fully funct
 Running LocalStack as a stand-in for AWS S3 during a test:
 
 ```java
-DockerImageName localstackImage = DockerImageName.parse("localstack/localstack:0.11.3");
+DockerImageName localstackImage = DockerImageName.parse("localstack/localstack:3.3.0");
 
 @Rule
 public LocalStackContainer localstack = new LocalStackContainer(localstackImage)
```

Comment:
Usually code snippets are provided from the test source code. Looks like this is not happening for LocalStack docs. Can you update the PR doing so?

Reply:
@sullis do you mind updating the PR?